src/popup/components/buttons/BtnClose.vue

```diff
@@ -16,8 +15,10 @@ export default {
   components: {
     BtnIcon,
   },
```

Comment:
Suggest to add lang="ts" in `script`.

Reply:
I believe this is out of the scope of this migration and can be done in a separate PR